mutexbot/action.yml

```diff
@@ -0,0 +1,23 @@
+# This file is generated by ghactions
+# Do not edit this file manually unless you disable the `generate` feature.
+
+name: Mutexbot
+description: Action to reserve and release resources on mutexbot.com
+inputs:
+  api_key:
+    description: Mutexbot API key
+  mode:
+    description: Whether to reserve or release or force-release
+    default: reserve
+  resource_name:
```

Comment:
```suggestion
  resource-name:
```

Reply:
Sorry, missed that part in the description:

> Parameters change from kebap-case to snake_case, because of rust best practices. I was not able to make rust happy with kebap-case in parameters. If should be possible to do that by patching https://github.com/42ByteLabs/ghactions